File: app/src/main/java/com/neome/api/home/aside/msg/MsgGroupMembersAdd.java
// neome.ai API - do not change
//

package com.neome.api.home.aside.msg;

import com.neome.api.core.base.msg.MsgVersion;
import com.neome.api.meta.base.Types.EntUserId;
import com.neome.api.meta.base.Types.GroupId;

import org.jetbrains.annotations.Nullable;

import java.util.Set;

@SuppressWarnings("unused")
public class MsgGroupMembersAdd extends MsgVersion
{
  public GroupId groupId;

  @Nullable
  public Set<EntUserId> insertAdminSet;

  @Nullable
  public Set<EntUserId> insertMemberSet;
}

File: app/src/main/java/com/neome/api/home/aside/msg/MsgGroupMembersAdd.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.aside.msg

import com.neome.api.core.base.msg.MsgVersion
import com.neome.api.meta.base.Types.EntUserId
import com.neome.api.meta.base.Types.GroupId

interface MsgGroupMembersAdd : MsgVersion {
    val groupId: GroupId
    val insertAdminSet: List<EntUserId>?
    val insertMemberSet: List<EntUserId>?
}
